resources/qml/UserProfile.qml

```diff
@@ -10,12 +10,19 @@ ApplicationWindow {
 
     property var profile
 
+    x: MainWindow.x + (MainWindow.width / 2) - (width / 2)
+    y: MainWindow.y + (MainWindow.height / 2) - (height / 2)
     height: 650
     width: 420
     minimumHeight: 420
     palette: colors
     color: colors.window
 
+    Shortcut {
+        sequence: "Esc"
```

Comment:
Should be QStandardKey.Cancel afaik

Reply:
This should still be QStandardKeys.Cancel. Otherwise this will be weird on macOS. https://doc.qt.io/qt-5/qkeysequence.html#standard-shortcuts

Reply:
Ah, didn't realize that there was StandardKeys.Cancel. Fixed.

Reply:
That's why you don't sneak in changes after the first review! D: